assets/app/styles/_spacers.less

```diff
@@ -0,0 +1,51 @@
+// Spacing utility classes
+// creates:
+//  .pad-sm, .pad-md, .pad-lg, etc
+//  .mar-sm, .mar-md, .mar-lg, etc
+//  .pad-left-sm, .mar-left-sm, etc
+//
+@nil: none 0 e('!important');
+// @auto 0 auto; is built into the fn below
+@xs: xs 3px e('');      // 2px
+@sm: sm 5px e('');      // 4px
+@md: md 10px e('');     // 8px
+@lg: lg 15px e('');     // 12px
+@xl: xl 20px e('');     // 16px
+@xxl: xxl 30px e('');   // 32px
```

Comment:
Yup, left my original vals in comments as we discussed in case I wanted/needed to return to them.  I'll squash this & remove them.